resources/css1.css

```diff
@@ -143,6 +154,7 @@ body{
 }
 .h2_about_section_2{
     margin-top: 160px;
+    font-weight: 800 !important;
```

Comment:
`!important` should not be used unless it is really necessary.
Maybe treat font-weights as separate classes along with font-family etc... Look at [How To Set Weights And Styles With The font-face Declaration](https://www.smashingmagazine.com/2013/02/setting-weights-and-styles-at-font-face-declaration/) for inspiration.